### DISCUSSION TITLE
What is a third space and why do I like the concept?

### DISCUSSION THREAD

**Discussion by giscus** [alifeee/blog#40] [comments]
# What is a third space and why do I like the concept?

I talk a lot about third spaces. These are some thoughts and opinions.

https://blog.alifeee.co.uk/what-is-a-third-space/

**Comment by somebody909** [comment]
The world needs more third spaces!

I know you have to pay for university, but I think theyre a good source of third spaces too. University students are spoilt for choice with third spaces, what with common rooms and student bars and student socieites etc. I wish the real world more of that.

It reminds me of this, one of the best tweets of all time:

***
Half the reason folks romanticize college is because it's the last time most folks lived in dense, walkable neighborhoods focused on providing community during plentiful off-hours.
***
https://twitter.com/NomeDaBarbarian/status/1374115572170956801
